wolox-books-vue/src/views/SignUp.vue

```diff
@@ -0,0 +1,84 @@
+<template lang="pug">
+  div.container.column.center.middle
```

Comment:
div can be removed

Reply:
this div sets the min-height to be 100vh so that the form is centered